src/test/java/seedu/address/logic/parser/CountryNoteAddCommandParserTest.java

```diff
@@ -35,23 +42,34 @@ public void parse_withPreamable_throwsParseException() {
     @Test
     public void parse_hasCountryNoNote_throwsParseException() {
         assertThrows(ParseException.class, () -> parser.parse(" c/SG random string"));
+        assertThrows(ParseException.class, () -> parser.parse(" c/SG t/random string"));
+        assertThrows(ParseException.class, () -> parser.parse(" c/SG t/"));
     }
 
     @Test
     public void parse_invalidCountryHasNote_throwsParseException() {
         assertThrows(ParseException.class, () -> parser.parse(" c/Z Z nt/random string"));
+        assertThrows(ParseException.class, () -> parser.parse(" c/Z Z t/a nt/random string"));
+        assertThrows(ParseException.class, () -> parser.parse(" c/Z Z t/ nt/"));
     }
 
     @Test
     public void parse_validCountryNoNote_throwsParseException() {
         assertThrows(ParseException.class, () -> parser.parse(" c/MY "));
+        assertThrows(ParseException.class, () -> parser.parse(" c/MY t/a"));
+        assertThrows(ParseException.class, () -> parser.parse(" c/MY nt/ t/"));
```

Comment:
```suggestion
        assertThrows(ParseException.class, () -> parser.parse(" c/MY nt/ t/a"));
        assertThrows(ParseException.class, () -> parser.parse(" nt/ c/MY t/a"));
```
Better isolation for testing no notes